src/components/Switch/Switch.js

```diff
@@ -56,15 +56,15 @@ const Switch = props => {
     return (
       <button {...other} {...commonProps}>
         {btnIcon}
-        {text}
+        <span>{text}</span>
```

Comment:
Is this part of the changes for X?

Reply:
yes it is meant to match [this line in vanilla](https://github.com/IBM/carbon-components/blob/master/src/components/content-switcher/content-switcher.hbs#L12)